tests/src/container-smoke.spec.ts

```diff
@@ -26,12 +26,13 @@ import { NavigationBar } from './model/workbench/navigation';
 import { waitUntil, waitWhile } from './utility/wait';
 import { deleteContainer, deleteImage } from './utility/operations';
 import { ContainerState } from './model/core/states';
+import type { ImagesPage } from './model/pages/images-page';
 
 let pdRunner: PodmanDesktopRunner;
 let page: Page;
-const imageToPull = 'quay.io/podman/hello';
+const imageToPull = 'docker.io/library/alpine';
 const imageTag = 'latest';
-const containerToRun = 'podman-hello';
+const containerToRun = 'alpine-container';
```

Comment:
docker.io is rate limited so are you sure we won't have more problems ?

Reply:
No. I just wanted an alternative to quay.io that seemed to have a couple of outages in a past month. What do you think a better option?

Reply:
if you want a more robust solution it would mean to start a local registry and pull from that local registry

but maybe you can try to pull from quay and then if it doesn't work you pull from docker.io

or you pull from github registry so if github is down anyway action will be down as well :-)

Reply:
Good idea! I found an alpine on ghcr that is publicly available. `ghcr.io/linuxcontainers/alpine:latest`